packages/tooltip/src/Tooltip.js

```diff
@@ -61,19 +62,21 @@ Tooltip.propTypes = {
   children: PropTypes.node,
   isVisible: PropTypes.bool,
   align: PropTypes.oneOf(values(alignments)),
+  position: PropTypes.oneOf(values(position)),
```

Comment:
Since we're already having a breaking change for `top`->`offset`, how about renaming the `align` and `position` props?
They're the same thing but for different axis. The naming doesn't make it clear which one is horizontal and which one is vertical.
We could go with something like `horizontalAlign`/`verticalAlign` or shorter, `hAlign`/`vAlign`. What do you think?

Reply:
It's better indeed, thank you! 
I would go with `hAlign`/`vAlign`, but what about `alignX`/`alignY`?

Reply:
I'll wait for the complete review before I do this change, ok?

Reply:
I like alignX/Y!